### ISSUE TITLE
Darknet issue trying to run slimyolov3

### ISSUE THREAD

**Issue by cartovarc** [ayooshkathuria/pytorch-yolo-v3#117]
I dowload this weights and cfg:
https://github.com/PengyiZhang/SlimYOLOv3

original tensor size: torch.Size([1, 3, 608, 608])

I got this error:

with map1 and map2 with sizes in darknet.py file:
torch.Size([1, 12, 2, 2])
torch.Size([1, 12, 3, 3])

  File "/home/aras/Desktop/rosas/savia/monitor/yolo/darknet.py", line 341, in forward
    x = torch.cat((map1, map2), 1)
RuntimeError: invalid argument 0: Sizes of tensors must match except in dimension 1. Got 3 and 2 in dimension 2

**Comment by cartovarc** [comment]
![Screenshot from 2019-09-03 20-45-02](https://user-images.githubusercontent.com/26529546/64219635-d1f85680-ce8b-11e9-97e2-8d76a1efe975.png)

**Comment by H-YunHui** [comment]
@cartovarc
have you solved it now?

**Comment by cartovarc** [comment]
No, I have to use another YOLO implemention and it works. This is the repo https://github.com/ultralytics/yolov3

**Comment by H-YunHui** [comment]
@cartovarc
thank you, I got it

**Comment by H-YunHui** [comment]
@cartovarc
I have a question about how to display each processed image for video stream processing?

**Comment by cartovarc** [comment]
Do you want stream to a browser ?

**Comment by H-YunHui** [comment]
@cartovarc
no, I just want to show it and save it, like the issue:
https://github.com/ultralytics/yolov3/issues/486

**Comment by H-YunHui** [comment]
@cartovarc
hi, I have a question would like to ask you, when you train the repo:
https://github.com/ultralytics/yolov3
 have you ever had this kind of problem like this issue:
https://github.com/ultralytics/yolov3/issues/479
